### ISSUE TITLE
ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7 failed

### ISSUE THREAD

**Issue by caesarxuchao** [kubernetes/kubernetes#48247]
The latest builds failed.

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7/34?log#log

```
google_compute_address.e2e-34-master-ip: Error creating address: googleapi: Error 403: Quota 'STATIC_ADDRESSES' exceeded. Limit: 2.0, quotaExceeded
```

@gmarek @mikedanese could you help take a look if its a test-infra problem or a real problem?

**Comment by luxas** [comment]
/assign @luxas @pipejakob 

@caesarxuchao I'm 99% sure this is infra related. We have had issues with quota before for the kubeadm jobs.

The periodic-1.7 job just experienced the same problem, but got up green after three failures, see: https://k8s-testgrid.appspot.com/sig-cluster-lifecycle#periodic-kubeadm-gce-1.7

So I think that's fine.

What I'd be more worried about is: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7/31

Regardless of whether that's kubeadm related or not; it's not a good sign if a job takes more than 2hrs more than expected ([compare with the previous one](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7/30)) and conformance tests are failing. I'll look into that more...

**Comment by luxas** [comment]
job 31 failed with 
```
Jun 29 07:06:49.867: All nodes should be ready after test, Not ready nodes: ", e2e-31-node-lxfp"
```

on every test. Is this a known flake?

**Comment by gmarek** [comment]
@luxas - it strongly depends on why nodes were not ready.

**Comment by gmarek** [comment]
Last two failures, as @luxas wrote are purely test-infra issues. I'll clean up the project.

**Comment by gmarek** [comment]
As for not ready Node - we're missing logs from Nodes, which means it's impossible to tell. I'd assume it's a flake for now, as it's highly unlikely that it's kubeadm related.

**Comment by luxas** [comment]
@gmarek Here's the failure: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/framework.go#L350

I don't think this is kubeadm-specific, but it's worth looking into.
The e2e's would never have run if all nodes weren't ready in the first place.
This happened while executing the test and seems like a timeout after 3 mins.

Do we know if the machines were at really heavy load at this time?
Or what can have caused 3h27min vs 1h12min?

IIRC there was some issues with namespace deletion or something earlier, is that fixed by now?
I guess so. This is on the v1.7 branch

**Comment by gmarek** [comment]
That's not the failure, it's just a report - check if Node is Ready. The failure happened on the Node. In the absence of logs it's pretty much impossible to tell why this happened.

3h27m vs 1h12 is probably cause by - waiting for machine to recover before/after every test. I guess the 3h27m ~timeout.

**Comment by luxas** [comment]
Ok, I think this is safe to move out from v1.7 after further digging.
Seems like we have some serious flakiness in the v1.6 branch
All these three jobs that ran release-1.6 e2e's against an release-1.6 control plane nearly at the same time, and flaked heavily:

(I got the data from here: https://k8s-testgrid.appspot.com/sig-cluster-lifecycle)

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/periodic-kubernetes-e2e-kubeadm-gce-1-6/769
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-6/117
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7/31

Only difference in the last test mentioning v1.7 (which this issue was opened for) is that the kubeadm CLI binary is v1.7. But the common denominator here indeed is v1.6 e2e's and control plane.

TL;DR; I think this can be moved out of v1.7, but indeed it's a severe test failure for v1.6

cc @caesarxuchao @ethernetdan @enisoc @dchen1107

**Comment by luxas** [comment]
> That's not the failure, it's just a report - check if Node is Ready

Yeah, indeed. Sorry for the clumsy wording, I meant "BTW; here's the source code line which yielded that message".
And yes, something disruptive must have happened to the Node while executing the e2e's that made it fail. Then all the other remaining tests also failed because they noticed that the node isn't healthy anymore.

**Comment by caesarxuchao** [comment]
Thanks for looking into this.

**Comment by pipejakob** [comment]
Since the newest run is completely [green](https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-gce-1-6-on-1-7/35), this definitely points at flakiness, but shouldn't block the 1.7 release. Should we open a separate issue to track down the source of flakes?

**Comment by luxas** [comment]
This is fixed. The job hasn't been flaky lately
